lib/core/versions/latest/models/SuffixDataModel.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal data structure of the delta for each operation.
+ */
+export default interface SuffixDataModel {
+  deltaHash: string;
```

Comment:
Just realized... why is this different color than the line below...

Reply:
There must be a reason, we just don't know.